pygmt/src/grdsample.py

```diff
@@ -0,0 +1,95 @@
+"""
+grdsample - Resample a grid onto a new lattice
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    J="projection",
+    I="increment",
```

Comment:
It seems other modules (e.g., `grdfilter`) use the alias `spacing` rather than `increment`. We should keep the alias consistent, no?